tests/refactored_tests_local.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+"""
+    MiniTwit Tests
+    ~~~~~~~~~~~~~~
+
+    Tests a MiniTwit application.
+
+    :refactored: (c) 2024 by dangr from HelgeCPH's refactored unittest version
+    :refactored: (c) 2024 by HelgeCPH from Armin Ronacher's original unittest version
+    :copyright: (c) 2010 by Armin Ronacher.
+    :license: BSD, see LICENSE for more details.
+"""
+import requests
+
+import unittest
+
+BASE_URL = "http://localhost:15000"
```

Comment:
This is the only reason to have a separate test file for running locally
If we weren't refactoring to go asap this would just be handled with an env var for the existing file.